mypy/checkstrformat.py

```diff
@@ -70,7 +70,7 @@ def check_str_interpolation(self, str: StrExpr, replacements: Node) -> Type:
         return self.named_type('builtins.str')
 
     def parse_conversion_specifiers(self, format: str) -> List[ConversionSpecifier]:
-        key_regex = r'(\((\w*)\))?'  # (optional) parenthesised sequence of characters
+        key_regex = r'(\(([\w ]*)\))?'  # (optional) parenthesised sequence of characters
```

Comment:
I decided to just change it. >_>
https://github.com/python/mypy/commit/7c766098824a7ea5ceaeb39cdcf5169d3cd75b9c